### PR TITLE
Fix misspelling at Fatal error Page

### LINKED ISSUE #1992: Wording on Fatal error screen
* "Suggested action to resolve *the* error"
* Rockchat -> *Rocketchat*
* Maybe add a link to https://www.decred.org/community/
* The "Please note" box seems misaligned 
image

### PR DESCRIPTION
**Pull request by vctt94**
Fixes #1992

### CODE REVIEW

File: app/components/views/FatalErrorPage.js [hunk 1]
```diff
@@ -38,7 +38,15 @@ class FatalErrorPage extends React.Component {
         </>);
       break;
     default:
-      errorMessage = <T id="fatal.suggestion.fallthrough" m="Please note the error above and go to the support channel on slack/matrix/rockchat for help resolving the issue." />;
+      errorMessage = (
+        <T id="fatal.suggestion.fallthrough"
+          m="Please note the error above and go to the support channel on slack/matrix/Rocketchat for help resolving the issue. {link}"
```
Comment (matheusd) [inline app/components/views/FatalErrorPage.js]: I'd remove slack and rocket chat since we give preference to matrix now.

Comment (vctt94) [reply app/components/views/FatalErrorPage.js]: I agree, so I added the following message: `Please note the error above and go to the support channel on matrix (via riot) or some other preferred chat channel for help resolving the issue.`

Comment (matheusd) [reply app/components/views/FatalErrorPage.js]: "(via riot)" is unnecessary. Users can choose whatever client they want.

Comment (vctt94) [reply app/components/views/FatalErrorPage.js]: Removed

**Pull request merged**